schema/bom-1.4-SNAPSHOT.proto

```diff
@@ -450,3 +452,179 @@ message Evidence {
+  optional string justification = 6;
+}
+
+enum Severity {
+  SEVERITY_CRITICAL = 0;
+  SEVERITY_HIGH = 1;
+  SEVERITY_MEDIUM = 2;
+  SEVERITY_LOW = 3;
+  SEVERITY_INFO = 4;
+  SEVERITY_NONE = 5;
+  SEVERITY_UNKNOWN = 6;
+}
+
+enum ScoreMethod {
+  // Common Vulnerability Scoring System v2 - https://www.first.org/cvss/v2/
+  SCORE_METHOD_CVSSV2 = 0;
+  // Common Vulnerability Scoring System v3 - https://www.first.org/cvss/v3-0/
+  SCORE_METHOD_CVSSV3 = 1;
+  // Common Vulnerability Scoring System v3.1 - https://www.first.org/cvss/v3-1/
+  SCORE_METHOD_CVSSV31 = 2;
+  // OWASP Risk Rating Methodology - https://owasp.org/www-community/OWASP_Risk_Rating_Methodology
+  SCORE_METHOD_OWASP = 3;
+  // Other scoring method
+  SCORE_METHOD_OTHER = 4;
+}
+
+message Advisory {
+  // An optional name of the advisory.
+  optional string title = 1;
+  // Location where the advisory can be obtained.
+  string url = 2;
+}
+
+message VulnerabilityCredits {
+  // The organizations credited with vulnerability discovery.
+  repeated OrganizationalEntity organizations = 1;
+  // The individuals, not associated with organizations, that are credited with vulnerability discovery.
+  repeated OrganizationalContact individuals = 2;
+}
+
+message VulnerabilityAnalysis {
+  // Declares the current state of an occurrence of a vulnerability, after automated or manual analysis.
+  optional ImpactAnalysisState state = 1;
+  // The rationale of why the impact analysis state was asserted.
+  optional ImpactAnalysisJustification justification = 2;
+  // A response to the vulnerability by the manufacturer, supplier, or project responsible for the affected component or service. More than one response is allowed. Responses are strongly encouraged for vulnerabilities where the analysis state is exploitable.
+  repeated VulnerabilityResponse response = 3;
+  // Detailed description of the impact including methods used during assessment. If a vulnerability is not exploitable, this field should include specific details on why the component or service is not impacted by this vulnerability.
+  optional string detail = 4;
+}
+
+enum ImpactAnalysisState {
+  // The vulnerability has been remediated.
+  IMPACT_ANALYSIS_STATE_RESOLVED = 0;
+  // The vulnerability has been remediated and evidence of the changes are provided in the affected components pedigree containing verifiable commit history and/or diff(s).
+  IMPACT_ANALYSIS_STATE_RESOLVED_WITH_PEDIGREE = 1;
+  // The vulnerability may be directly or indirectly exploitable.
+  IMPACT_ANALYSIS_STATE_EXPLOITABLE = 2;
+  // The vulnerability is being investigated.
+  IMPACT_ANALYSIS_STATE_IN_TRIAGE = 3;
+  // The vulnerability is not specific to the component or service and was falsely identified or associated.
+  IMPACT_ANALYSIS_STATE_FALSE_POSITIVE = 4;
+  // The component or service is not affected by the vulnerability. Justification should be specified for all not_affected cases.
+  IMPACT_ANALYSIS_STATE_NOT_AFFECTED = 5;
+}
```

Comment:
As above, IMPACT_ANALYSIS_STATE_NULL as default 0 value.

Reply:
Changed in 6201ede37ac9c730acfe414073fa281ab900ad98